component/receiver.go

```diff
@@ -24,10 +24,7 @@ import (
 // ReceiverConfig is the configuration of a component.Receiver. Specific extensions must implement
```

Comment:
```suggestion
// ReceiverConfig is the configuration of a component.Receiver. Specific receivers must implement
```

Reply:
Done separately since not related to this PR. https://github.com/open-telemetry/opentelemetry-collector/pull/6551